### PR TITLE
Add quickstart section to docs.

### PR DESCRIPTION
**Pull request by OlgaMaciaszek**

### CODE REVIEW

**Review by spencergibb — APPROVED** [review]

File: docs/src/main/asciidoc/quickstart.adoc [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+      <plugin>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-maven-plugin</artifactId>
+      </plugin>
+    </plugins>
+  </build>
+</project>
+----
+
+The example below shows a typical Gradle setup:
```
Comment (Buzzardo) [inline docs/src/main/asciidoc/quickstart.adoc]: Change "The example below" to "The following example".

File: docs/src/main/asciidoc/quickstart.adoc [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+    }
+
+    public static void main(String[] args) {
+        SpringApplication.run(Application.class, args);
+    }
+
+}
+----
+
+When this HTTP server runs, it connects to Consul Agent running at the default local 8500 port.
```
Comment (Buzzardo) [inline docs/src/main/asciidoc/quickstart.adoc:101]: Change "the default local 8500 port." to "the default local port: 8500."

File: docs/src/main/asciidoc/quickstart.adoc [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+To modify the startup behavior, you can change the location of Consul Agent by using `application.properties` as shown in the following example:
```
Comment (Buzzardo) [inline docs/src/main/asciidoc/quickstart.adoc]: Add a comma before "as shown".

File: docs/src/main/asciidoc/quickstart.adoc [hunk 4]
```diff
@@ -0,0 +1,224 @@
...
+
+----
+spring:
+  cloud:
+    consul:
+      host: localhost
+      port: 8500
+----
+
+You can now use `DiscoveryClient`, `@LoadBalanced RestTemplate` or `@LoadBalanced WebClient.Builder` to retrieve services and instances data from Consul, as shown in the following example:
```
Comment (Buzzardo) [inline docs/src/main/asciidoc/quickstart.adoc]: Add a comma after "`@LoadBalanced RestTemplate`".

File: docs/src/main/asciidoc/quickstart.adoc [hunk 5]
```diff
@@ -0,0 +1,224 @@
...
+        return list.get(0).getUri().toString();
+    }
+    return null;
+}
+----
+
+=== Distributed Configuration Usage
+
+To use these features in an application, you can build it as a Spring Boot application that depends on `spring-cloud-consul-core` and `spring-cloud-consul-config`.
+The most convenient way to add the dependency is with a Spring Boot starter `org.springframework.cloud:spring-cloud-starter-consul-config`.
```
Comment (Buzzardo) [inline docs/src/main/asciidoc/quickstart.adoc]: Add a colon (:) after "starter".

File: docs/src/main/asciidoc/quickstart.adoc [hunk 6]
```diff
@@ -0,0 +1,224 @@
...
+      <plugin>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-maven-plugin</artifactId>
+      </plugin>
+    </plugins>
+  </build>
+</project>
+----
+
+The example below shows a typical Gradle setup:
```
Comment (Buzzardo) [inline docs/src/main/asciidoc/quickstart.adoc]: Change "The example below" to "The following example".

File: docs/src/main/asciidoc/quickstart.adoc [hunk 7]
```diff
@@ -0,0 +1,224 @@
...
+    }
+
+    public static void main(String[] args) {
+        SpringApplication.run(Application.class, args);
+    }
+
+}
+----
+
+The application will retrieve configuration data from Consul.
```
Comment (Buzzardo) [inline docs/src/main/asciidoc/quickstart.adoc]: Change "will retrieve" to "retrieves".

**Review by Buzzardo — CHANGES_REQUESTED** [review]

**Review by Buzzardo — APPROVED** [review]

**Pull request merged**